core/app/models/spree/stock/simple_coordinator.rb

```diff
@@ -20,41 +20,64 @@ module Stock
     # Finally, shipping rates are calculated using the class configured as
     # Spree::Config.stock.estimator_class.
     class SimpleCoordinator
-      attr_reader :order
+      attr_reader :order, :inventory_units, :splitters, :stock_locations,
+        :filtered_stock_locations, :inventory_units_by_variant, :desired,
+        :availability, :allocator, :packages
```

Comment:
```suggestion
      attr_reader :order
      
      # This class is notoriously opaque. To aid in debugging, we've provided
      # the following methods. These are not considered part of the public API
      # of this class and may be removed without deprecation in a future
      # version of Solidus. Do not use them in production code.
      attr_reader :inventory_units, :splitters, :stock_locations,
        :filtered_stock_locations, :inventory_units_by_variant, :desired,
        :availability, :allocator, :packages
```

Reply:
`@api private` should also do the trick :slightly_smiling_face:

Reply:
🙏🏻 Perfect!

Reply:
@BenMorganIO did you see this comment about using `@api private`?

Reply:
@BenMorganIO, what do you think about just using `@api private`? This one is very close to being merged :slightly_smiling_face:

Reply:
nvm, I did it myself :slightly_smiling_face: 

@jarednorman, are you ok with merging it now?